src/text/raw_text_writer.rs

```diff
@@ -32,7 +32,7 @@ impl RawTextWriterBuilder {
     /// ```
     pub fn compact() -> RawTextWriterBuilder {
         RawTextWriterBuilder {
-            whitespace_config: COMPACT_WHITESPACE_CONFIG.clone(),
+            whitespace_config: COMPACT_WHITESPACE_CONFIG,
```

Comment:
🗺️ These types don't own any heap data, so they're now `Copy`.

Reply:
The various `*_CONFIG` are constants that could just be borrowed, right?

Reply:
It was actually already `Box`ed in the `RawTextWriter`, it was only inline in the builder so the fields could be modified. 😅 

Either way, this builder and its writer are being deleted in the next PR. 😛